plugin/pkg/admission/limitranger/interfaces.go

```diff
@@ -14,14 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package limitranger
+package limitrangeradmission
```

Comment:
please revert this change

Reply:
The rename was needed to prevent failure of this golint rule:
```
Errors from golint:
plugin/pkg/admission/limitranger/interfaces.go:26:6: type name will be used as limitranger.LimitRangerActions by other packages, and that stutters; consider calling this Actions
```
I'm open to suggestions on fixing it. The original approach keeps the same use in all clients and just renames it internally.

Reply:
I'm not in favor of changing our package and type names to satisfy that lint recommendation

Reply:
One option would be to rename the type, like the linter suggests, from LimitRangerActions to Actions. Another option is to keep this one blacklisted and unchecked until a future time so we can all agree on a valid solution. Which would you prefer?

Reply:
i'm fine w/ both. it's just the smaller change looks more happy.

Reply:
I've removed the renaming with its effects on all files and restored the blacklist on the limitranger package. I've kept all other linting fixes so it's a smaller effort once there's an agreement on what to do for this particular issue.

Reply:
> I've removed the renaming with its effects on all files and restored the blacklist on the limitranger package. I've kept all other linting fixes so it's a smaller effort once there's an agreement on what to do for this particular issue.

sounds good, thanks